projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -75,3 +99,18 @@ export const enum TitlePosition {
   Header = 'header',
   Footer = 'footer'
 }
+
+export const enum TitleStyle {
+  Regular = 'regular',
+  GrayedOut = 'grayed-out'
+}
+
+export const enum LinkButtonStyle {
+  Primary = 'primary',
```

Comment:
why only primary and quaternary? what happened to Secondary and Tertiary?

Reply:
if this is supposed to be equivalent to `ButtonRole`, we might as well use that directly in this component.

Reply:
Button Role removed